internal/app/http/service.go

```diff
@@ -113,7 +113,7 @@ func (dhs *defaultHandlerService) ListMpns(ctx context.Context, search *model.Se
 
 func (dhs *defaultHandlerService) FindInventoryEntry(ctx context.Context, name string) (*model.FoundEntry, error) {
 	//todo: check if name is valid format
-	toc, err := dhs.ListInventory(ctx, &model.SearchParams{Name: name})
+	toc, err := dhs.ListInventory(ctx, &model.SearchParams{Name: name, Options: model.SearchOptions{NameFilterType: model.FullMatch}})
```

Comment:
I think this Options FullMatch is just an explicit hint for developers, as FullMatch should be default anyway, right ?

Reply:
Correct. I decided to make it explicit here